backend/app/cmd/server.go

```diff
@@ -354,7 +354,7 @@ func (s *ServerCommand) HandleDeprecatedFlags() (result []DeprecatedFlag) {
 		s.ImageProxy.HTTP2HTTPS = s.LegacyImageProxy
 		result = append(result, DeprecatedFlag{Old: "img-proxy", New: "image-proxy.http2https", Version: "1.5"})
 	}
-	if len(s.Notify.Type) != 0 && (len(s.Notify.Users) != 0 || len(s.Notify.Admins) != 0) {
+	if len(s.Notify.Type) != 0 && !(len(s.Notify.Type) == 1 && contains("none", s.Notify.Type)) && (len(s.Notify.Users) != 0 || len(s.Notify.Admins) != 0) {
```

Comment:
this is confusing. I would guess this `!(len(s.Notify.Type) == 1` means `len(s.Notify.Type) != 1` and if so, both first lens can be `len() > 1` ? I would be also nice to add a comment here, because just from this part it is unclear what is so special about a single element s.Notify.Type slice.

Reply:
I've added comments about each part of the expression, please take a look.